code/modules/atmospherics/machinery/pipes/heat_exchange/he_pipes.dm

```diff
@@ -1,6 +1,6 @@
 /obj/machinery/atmospherics/pipe/heat_exchanging
 	level = 2
-	var/minimum_temperature_difference = 20
+	var/minimum_temperature_difference = 0.01
```

Comment:
IMO having it be `1` would be a more sensible balance between performance and usefulness.

Reply:
I agree that 1 wuold probably be better

Reply:
```suggestion
	var/minimum_temperature_difference = 1
```